group_vars/all/organizations.yml

```diff
@@ -3,6 +3,7 @@ controller_organizations_all:
   - name: config_as_code
     galaxy_credentials:
       - ah_published
+      - ah_validated
```

Comment:
while it doesn't matter currently the order if you could put it below certified please because that is the correct order to have your list in (order does matter in theory) once they add the feature to be able to set the order by the module